core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -360,25 +365,59 @@
           + "logs over this threshold is minor compacted.  Also any tablet referencing this many "
           + "logs or more will be compacted."),
+  TSERV_WAL_MAX_SIZE("tserver.wal.max.size", "1G", PropertyType.BYTES,
+      "The maximum size for each write-ahead log. See comment for property"
+          + " tserver.memory.maps.max"),
+  @Deprecated(since = "2.1.0")
+  @ReplacedBy(property = Property.TSERV_WAL_MAX_SIZE)
   TSERV_WALOG_MAX_SIZE("tserver.walog.max.size", "1G", PropertyType.BYTES,
       "The maximum size for each write-ahead log. See comment for property"
           + " tserver.memory.maps.max"),
+  TSERV_WAL_MAX_AGE("tserver.wal.max.age", "24h", PropertyType.TIMEDURATION,
+      "The maximum age for each write-ahead log."),
+  @Deprecated(since = "2.1.0")
+  @ReplacedBy(property = Property.TSERV_WAL_MAX_AGE)
   TSERV_WALOG_MAX_AGE("tserver.walog.max.age", "24h", PropertyType.TIMEDURATION,
       "The maximum age for each write-ahead log."),
+  TSERV_WAL_TOLERATED_CREATION_FAILURES("tserver.wal.tolerated.creation.failures", "50",
+      PropertyType.COUNT,
+      "The maximum number of failures tolerated when creating a new write-ahead"
+          + " log. Negative values will allow unlimited creation failures. Exceeding this"
+          + " number of failures consecutively trying to create a new write-ahead log"
+          + " causes the TabletServer to exit."),
+  @Deprecated(since = "2.1.0")
+  @ReplacedBy(property = Property.TSERV_WAL_TOLERATED_CREATION_FAILURES)
   TSERV_WALOG_TOLERATED_CREATION_FAILURES("tserver.walog.tolerated.creation.failures", "50",
       PropertyType.COUNT,
       "The maximum number of failures tolerated when creating a new write-ahead"
           + " log. Negative values will allow unlimited creation failures. Exceeding this"
           + " number of failures consecutively trying to create a new write-ahead log"
           + " causes the TabletServer to exit."),
+  TSERV_WAL_TOLERATED_WAIT_INCREMENT("tserver.wal.tolerated.wait.increment", "1000ms",
+      PropertyType.TIMEDURATION,
+      "The amount of time to wait between failures to create or write a write-ahead log."),
+  @Deprecated(since = "2.1.0")
+  @ReplacedBy(property = Property.TSERV_WAL_TOLERATED_WAIT_INCREMENT)
   TSERV_WALOG_TOLERATED_WAIT_INCREMENT("tserver.walog.tolerated.wait.increment", "1000ms",
       PropertyType.TIMEDURATION,
       "The amount of time to wait between failures to create or write a write-ahead log."),
   // Never wait longer than 5 mins for a retry
+  TSERV_WAL_TOLERATED_MAXIMUM_WAIT_DURATION("tserver.wal.maximum.wait.duration", "5m",
+      PropertyType.TIMEDURATION,
+      "The maximum amount of time to wait after a failure to create or write a write-ahead log."),
+  // Never wait longer than 5 mins for a retry
+  @Deprecated(since = "2.1.0")
+  @ReplacedBy(property = Property.TSERV_WAL_TOLERATED_MAXIMUM_WAIT_DURATION)
```

Comment:
Although the Property names are not sorted, we usually try to group common ones together. You could put the new tserver ones all together with the other "tserver.wal" properties. But I don't think this is a big deal since they can be sorted in an IDE.

Reply:
@milleruntime I'm not sure I understand your comment. It looks like these are already grouped together. Each new one looks like it was added directly next to the one it replaced, which is a nice, minimal change.